### ISSUE TITLE
accuracy issue

### ISSUE THREAD

**Issue by ashleycolton** [shokiami/GeoKnowr#1]
I have tested a few images, and the results is not accurate.. not sure if it's the algo.
for example, 
I fed an image of an island in spain, it gave coordinates of brazil.
feed image of norway, it says australia.

I used this 
playwright
scikit-learn==1.1.3
pandas
matplotlib
+torch&torchvision

Gaussian mixture model prompted I need to use 1.1.3 for more reliable performance, but the output is the same location.
very interesting project, you took a different approach compared to pigeon, maybe training on larger dataset could really benefit in the precision!
